content/docs/command-reference/metrics/index.md

```diff
@@ -18,15 +18,15 @@ positional arguments:
 ## Description
 
 In order to follow the performance of machine learning experiments, DVC has the
-ability to mark stage <abbr>outputs</abbr> or other files as metrics. These
-metrics are project-specific floating-point or integer values e.g. AUC, ROC,
-false positives, etc.
-
-In <abbr>pipelines</abbr>, metrics files are typically generated by user data
-processing code, and are tracked using the `-m` (`--metrics`) and `-M`
-(`--metrics-no-cache`) options of `dvc stage add`. If using
-[DVCLive](/doc/dvclive/live/log_metric), the files are generated and tracked
-automatically. Metrics files may also may be manually added to
+ability to mark [structured files](#supported-file-formats) containing key/value
+pairs as metrics. These metrics are project-specific floating-point, integer, or
+string values e.g. AUC, ROC, false positives, etc.
+
+If using [DVCLive](/doc/dvclive/live/log_metric), the files are generated and
+tracked automatically. Otherwise, in <abbr>pipelines</abbr>, metrics files are
```

Comment:
if reads a bit like "Otherwise [if you don't use DVCLive] ..." - I think we have cases where I could use both, though?

also, in the first sentence - "tracked automatically" - does it mean by Git? or what exactly?

Reply:
may be we should just a separate section - using metrics with DVC pipelines? in all of those docs ... otherwise we complicate the easy path and also can't explain actually what's going and why ..

Reply:
Okay, let's just drop it here at least and follow up on removing all the metrics/plots stage outputs.